rose/common/actions.py

```diff
@@ -1,10 +1,10 @@
 """ Driving actions """
```

Comment:
Just for consistency, we can omit the spaces here but I don't mind leaving it

Reply:
If black did not change it it is fine.